### ISSUE TITLE
New Logo?

### ISSUE THREAD

**Issue by anaufalm** [0xFireball/PluckLockEx#4]
Hi, Im freelance graphic designer. 

I just saw your app, and unfortunately there is no interesting logo yet. I want to contribute to your project, I can create a new logo for your app. What do you think? :)

Rgrd,
Naufal

[My Recent portfolio](https://www.instagram.com/faal.id/)

**Comment by 0xFireball** [comment]
Hello,
Your logos are very impressive! The current logo for the project is just a placeholder, and you are welcome to design the new logo, all help is appreciated :)
Thanks for your interest in the project!

**Comment by anaufalm** [comment]
Let's explain briefly about your logo, like what do you want or you already have an idea for it? So, we can make it better.

**Comment by 0xFireball** [comment]
Perhaps something like a phone being pulled away or grabbed, but without an actual hand, so something conveying motion?

**Comment by anaufalm** [comment]
Okay, got it.
You wanna [smartphone](https://www.google.co.id/search?biw=1366&bih=700&tbm=isch&sa=1&ei=AFJXW8fqA8rWvgSrkpe4AQ&q=smartphone+icon&oq=smartphone+icon&gs_l=img.3..0i67k1j0j0i7i30k1l3j0i30k1l5.14117.15201.0.15503.5.5.0.0.0.0.222.260.1j0j1.2.0....0...1c.1.64.img..3.2.259...0i7i10i30k1j0i7i5i30k1j0i8i7i30k1.0.1gXD8m_Xx_M) or just a [phone](https://www.google.co.id/search?biw=1366&bih=700&tbm=isch&sa=1&ei=-FFXW-zaCcX_vgTr_bboAQ&q=phone+icon&oq=phone+icon&gs_l=img.3..0l10.4083.6542.0.6825.5.5.0.0.0.0.259.753.2-3.3.0....0...1c.1.64.img..2.3.751...35i39k1j0i67k1.0.3XZ9NrKzGKY)?

**Comment by 0xFireball** [comment]
Smartphone probably matches this app better.

**Comment by anaufalm** [comment]
What about this?
Is there any idea or improvement feel free to comment :)

![plx1](https://user-images.githubusercontent.com/35919168/43160200-b5f98aac-8fae-11e8-8d27-c386854b1a65.png)

![plx2](https://user-images.githubusercontent.com/35919168/43160206-ba95fcf8-8fae-11e8-89e4-ff9675a57fcc.png)

**Comment by 0xFireball** [comment]
Hi,
Thanks for your logos!
I particularly liked the motion effect in this one of yours: https://www.instagram.com/p/BkN7pC-AFtf/?taken-by=faal.id
perhaps, something like that but instead of the "drops" the trail in the ones you designed above.
I think 3 trail items of the same length instead of 4-5
For the phone, could we use a more minimalist icon? Maybe even a rectangle outline?
Also, I like the solid background-colored logos.
Could the phone be straight instead of slanted?

Thanks again for your contribution!

**Comment by anaufalm** [comment]
Great, I will try

**Comment by anaufalm** [comment]
How looks?

![plx3](https://user-images.githubusercontent.com/35919168/43274330-251213a2-9129-11e8-9339-8142c550f71f.png)

**Comment by 0xFireball** [comment]
![image](https://user-images.githubusercontent.com/15932100/43277417-d71a2070-90bc-11e8-9e3a-edb9e7605492.png)

This one's my favorite, could the trails be a little thinner?

**Comment by anaufalm** [comment]
Yeah sure, but if you keep the trail only 3 it will look very small and looks weird.

What if I make it five trail? What do you think?
Like this:

![plx4](https://user-images.githubusercontent.com/35919168/43279467-35fdd5c0-9138-11e8-91b4-9a0ce33b2a23.png)

**Comment by 0xFireball** [comment]
Yes, I agree. Bottom left. Could the trails be made so they are a little bit faded compared to the phone? Otherwise this looks great!

**Comment by anaufalm** [comment]
Thanks, you mean trails and phone are made in different colors? or made some sort of gradient?

**Comment by 0xFireball** [comment]
slightly different colors, faded maybe

**Comment by anaufalm** [comment]
Like this? Which one do you want?

![plx5](https://user-images.githubusercontent.com/35919168/43282570-aea30b50-9140-11e8-8f15-277d853874c3.png)

**Comment by naufaal** [comment]
Hi @0xFireball, this is me anaufalm. Im sorry, I just lost my account. Can we continue this logo?
